runtime/src/main/java/org/corfudb/runtime/object/transactions/OptimisticTransactionalContext.java

```diff
@@ -323,7 +323,7 @@ protected void tryCommitAllProxies() {
         // First, get the committed entry
         // in order to get the backpointers
         // and the underlying SMREntries.
-        ILogData committedEntry = this.builder.getRuntime()
+        ILogData committedEntry = this.transaction.getRuntime()
```

Comment:
Why not to pass `CorfuRuntime` in a constructor as a parameter?

Reply:
Just renaming, not changing the logic of tryCommitAllProxies